### DISCUSSION TITLE
Breakpoint continuation

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#15481] [Q&A] [answered]
Based on Minio,Is there any recommended scheme to support breakpoint continuous transmission？

**Comment by vadmeste** [comment, marked as answer]
@XTUxiongda do you mean object multipart upload ? if yes, then that's supported.

For reference, https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html

**Reply by XTUxiongda** [reply]
https://help.aliyun.com/document_detail/407653.html
For example, Alibaba cloud's breakpoint continuous upload function

**Reply by harshavardhana** [reply]
We are S3 spec not Alibaba

**Reply by XTUxiongda** [reply]
I looked at the implementation of Java SDK. It will be uploaded automatically in blocks and then merged。
If there is a fault in the uploading process, what is the recommended solution?
Or can I only upload the whole big file to overwrite the old one?
